### DISCUSSION TITLE
The build has an error.

### DISCUSSION THREAD

**Discussion by furuhashitcb** [codemagic-ci-cd/codemagic-docs#999] [Q&A] [answered]
What are the possible factors?
----------------------------
Build failed :|
Failed to install dependencies
Could not find a file named "pubspec.yaml" in "/Users/builder/clone".
----------------------------

**Comment by himesh-cm** [comment, marked as answer]
Hi @furuhashitcb 
Please check if the file `pubspec.yaml` is available in your project and committed to git before your run it on Codemagic.

**Reply by furuhashitcb** [reply]
Thank you very much for your prompt reply.
I think that it will take time as a Flutter beginner. I will confirm it.
